sklearn/preprocessing/tests/test_data.py

```diff
@@ -2683,3 +2690,19 @@ def test_power_transformer_constant_feature(standardize):
             assert_allclose(Xt_, np.zeros_like(X))
         else:
             assert_allclose(Xt_, X)
+
+
+@pytest.mark.parametrize(
+    "array_namespace, device, dtype", yield_namespace_device_dtype_combinations()
+)
+def test_standard_scaler_array_api_compliance(array_namespace, device, dtype):
+    xp, device, dtype = _array_api_for_tests(array_namespace, device, dtype)
+
+    from sklearn.datasets import make_classification
```

Comment:
This should either be at the top of the file or we can just use something like `random_state.randn(n_samples, n_features)`.